public_html/wp-content/plugins/wordcamp-speaker-feedback/views/form-feedback.php

```diff
@@ -7,8 +7,51 @@
 <form id="sft-feedback" class="speaker-feedback">
 	<h3><?php esc_html_e( 'Rate this talk', 'wordcamporg' ); ?></h3>
 	
-	<div>
-		star rating…
+	<div class="speaker-feedback__field">
+		<fieldset class="speaker-feedback__field-rating" aria-label="<?php esc_attr_e( 'Rate this talk', 'wordcamporg' ); ?>">
+			<input
+				type="radio"
+				name="feedback-rating"
+				value="0"
+				checked
+				aria-label="<?php esc_attr_e( 'No stars', 'wordcamporg' ); ?>"
+			/>
+			<input type="radio" id="sft-rating-val-1" name="feedback-rating" value="1" />
+			<label for="sft-rating-val-1">
+				<span class="screen-reader-text"><?php esc_html_e( '1 star', 'wordcamporg' ); ?></span>
+				<svg xmlns="http://www.w3.org/2000/svg" viewbox="0 0 20 20" height="20" width="20" role="img" aria-hidden="true">
+					<path d="M10 1l3 6 6 .75-4.12 4.62L16 19l-6-3-6 3 1.13-6.63L1 7.75 7 7z"/>
+				</svg>
```

Comment:
Would it be worth putting this in an include file or variable, since it's the same thing five times? I could see us reusing it elsewhere to display an average rating or something.

Reply:
I've updated this to `require get_assets_path() . 'svg/star.svg';` , which should work fine as long as we [don't add in the XML header tag.](https://sheelahb.com/blog/how-to-get-php-to-play-nicely-with-svg-files/)

Reply:
Could it just be in an image tag instead?

Reply:
No, because it uses CSS to change the fill color, and we can't do that unless the SVG is inlined

Reply:
Ah, ok. 👍